group_vars/all

```diff
@@ -0,0 +1,16 @@
+# Variables for the group staging
+user: hiseq.bioinfo
+user_lims: glsai
+data_dir: /srv/illumina
+
+# Cruddy code ahoy!
+miniconda_path: '~/.miniconda'
+main_venv: master
+sw_path: /home/{{ user }}/opt
```

Comment:
I see this `sw_path` is also used for `taca` role. But currently we don't clone repos individually to each `NAS`/`preproc` (though not entirely sure about the preproc). We have the repo cloned in `mfs` filesystem which is mounted on all `NAS/preproc`, so would save the trouble off pulling the changes in all the servers while updating it.

Reply:
For preproc sw has always been cloned to ~/opt/... . Not sure about the NASes though.
The NAS recipe is a WIP since most things have been written in the dark.